document/js-api/index.bs

```diff
@@ -1327,14 +1329,12 @@ WebAssembly Module Records have the following methods:
 <h3 id="module-execution">ExecuteModule ( [ |promiseCapability| ] ) Concrete Method</h3>
 1. Assert: |promiseCapability| was provided.
 1. Let |record| be this WebAssembly Module Record.
-1. Assert: |record|.\[[Async]] is true.
-1. Assert: |record|.\[[ModuleAsync]] is true.
+1. Assert: |record|.\[[HasTLA]] is true.
 1. Let |module| be |record|.\[[WebAssemblyModule]].
```

Comment:
Should we change this to `[[ModuleSourceObject]]`?

Reply:
Given that the exact semantics of how import reflection interacts with host specs are still changing, I would leave this PR as-is (which already makes the current text compatible with the current ECMA-262), and we can iterate on that once we have the final design for the import reflection proposal.